net/openconnect/files/openconnect.sh

```diff
@@ -65,16 +66,25 @@ proto_openconnect_setup() {
 	}
 	[ -n "$authgroup" ] && append cmdline "--authgroup $authgroup"
 	[ -n "$username" ] && append cmdline "-u $username"
-	[ -n "$password" ] && {
+	[ -n "$password" ] || [ "$token_mode" = "script" ] && {
 		umask 077
 		mkdir -p /var/etc
 		pwfile="/var/etc/openconnect-$config.passwd"
-		echo "$password" > "$pwfile"
-		[ -n "$password2" ] && echo "$password2" >> "$pwfile"
+		[ -n "$password" ] && {
+			echo "$password" > "$pwfile"
+			[ -n "$password2" ] && echo "$password2" >> "$pwfile"
+		}
+		[ "$token_mode" = "script" ] && {
+			$token_script > "$pwfile" 2> /dev/null || {
+				logger -t openconenct "Cannot get password from script '$token_script'"
+				sleep 60
```

Comment:
Why such a long sleep?

Reply:
@nmav It is to prevent failing too fast and overload the router... I added here because I don't find how to define retry interval in netifd globally. Should I remove it?